deployments/tyk/README.md

```diff
@@ -85,3 +85,46 @@ The bootstrap script sends requests for each scenario, which you can see in the
 During the bootstrap script, the Gateway is used to create a plugin bundle, which it signs with its private key so that it can be validated before the Gateway loads it. The plugin is then copied to an Apache server to make it available to the Gateway at runtime when the *Python Middleware API* is requested.
 
 The source and manifest file for the plugin are available in `deployments/tyk/volumes/tyk-gateway/middleware/python/basic-example`.
+
+### JWT scopes example
+
+This example is using elliptic curve keys.
+It also demo the usage of scopes
+The scope claim name is `"aud"` and the expected scopes are `"client_id_x-readonly" ` and `"client_id_x-readwrite" `. 
+If you have one of these claims then you use the `jwks RO policy readonly` or `jwks RW policy readonly`. 
```

Comment:
Should 'readonly' be removed from these Policy names? They already contain 'RO' and 'RW'.